src/server/ttl/ttl_compaction_filter.rs

```diff
@@ -11,9 +11,23 @@ use engine_rocks::{
     RocksTtlProperties,
 };
 use engine_traits::raw_ttl::ttl_current_ts;
+use prometheus::*;
 
 use crate::server::metrics::TTL_CHECKER_ACTIONS_COUNTER_VEC;
 
+lazy_static! {
+    pub static ref TTL_EXPIRE_KV_SIZE_COUNTER: IntCounter = register_int_counter!(
+        "tikv_ttl_expire_kv_size_total",
```

Comment:
nit: would it make sense to use ``raw`` in the string name? e.g., "``ikv_ttl_raw_expire_kv_{size,entry}_total``

Reply:
The other metrics in `TTL` navigation are named as tikv_ttl_*, like https://github.com/tikv/tikv/blob/master/src/server/metrics.rs#L365, maybe we should keep them consistent